provider/repodata/data/json/props/names/gcm@1/und.json

```diff
@@ -0,0 +1,84 @@
+{
+  "map": {
+    "C": 65530,
```

Comment:
Thought/Discussion: should gcm contain both the single-value and the multi-value masks? Because then we're duplicating (a small amount of) data.